docs/update-notes.rst

```diff
@@ -9,6 +9,80 @@ It lists changes to technical requirements for new versions and additional instr
+----------------------------------------------
+Signavio Workflow has been renamed to Signavio Workflow Accelerator.
+If you use Workflow Accelerator in combination with Signavio Process Manager, you must update Process Manager to version 10.15.
+
+.. important:: 
+	To update Workflow Accelerator from a version prior to 3.11, you MUST first update the system to version 3.15 before you can update to version 3.22. 
+	Download `Signavio Workflow 3.15 <https://download.signavio.com/onpremise/latest/workflow/signavio-workflow-v3.15.11.zip>`_ and follow the respective previous update instructions, before you continue with this section.
+
+Besides the application update you are also advised to update your MongoDB instance.
+See the following section for more information.
+
+Updating MongoDB
+````````````````
+Workflow Accelerator 3.22 now supports the latest version of MongoDB (3.4.x) which comes with improved performance and functionality.
+You are advised to update your instance of MongoDB to version 3.4.
+Future releases of Workflow Accelerator will use the extended functionality and assume version 3.4 is in use.
+
+You can only update to MongoDB 3.4 if you are currently using version 3.2.
+The official MongoDB documentation contains for more information about the update:
+https://docs.mongodb.com/manual/release-notes/3.4-upgrade-standalone/
+
+To update your standalone MongoDB 3.2 to version 3.4 follow these steps:
+
+#. Stop the Tomcat application server - see :ref:`stopping`.
+#. Create a backup of your database using ``mongodump`` - see :ref:`backup`.
+#. Stop MongoDB - see :ref:`stopping`.
+#. *Windows only:* manually remove the created MongoDB service by executing ``$MONGO_HOME\bin\mongod.exe --remove`` with administrator privileges on the command line ``cmd``
+#. Uninstall MongoDB 3.2.x, but ensure you don't delete the data directory and configuration file of your database as well as the backup!
+#. Install MongoDB 3.4.x.
+
+	* You can find a suitable version for your OS in the MongoDB download center: https://www.mongodb.com/download-center
+
+#. *Windows only:* Open the command line ``cmd`` with administrative privileges and execute the following command. Make sure to use absolute paths and replace them with the matching ones on your system.
+
+    * ``C:\MongoDB\bin\mongod.exe --config C:\MongoDB\mongod.cfg --install``
+    * The referenced configuration file is the one you have created previously for the old version of MongoDB and should still be in the same location.
+
+#. Start the new MongoDB instance.
+#. Enable new functionality for version 3.4 by executing the following command on command line
+
+Windows
+~~~~~~~
+#. Open a command line with administrative privileges
+#. Navigate to the MongoDB bin directory
+
+	* For example ``C:\MongoDB\bin``
+
+#. Execute the following command:
+
+	* Replace ``<enterYourUserHere>`` with your MongoDB user, e.g. ``signavio``
+	* Replace ``<enterYourPasswordHere>`` with your MongoDB user password
```

Comment:
Hmm yeah, although this version looks more disruptive and not like it belongs to the command. :D